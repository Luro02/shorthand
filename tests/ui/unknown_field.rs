use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand(enable(garbage))]
#[shorthand(disable(garbage))]
pub struct Command {
    value: String,
}

fn main() {}
