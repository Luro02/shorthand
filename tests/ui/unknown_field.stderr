error: unknown field `garbage`
 --> $DIR/unknown_field.rs:4:20
  |
4 | #[shorthand(enable(garbage))]
  |                    ^^^^^^^

error: unknown field `garbage`
 --> $DIR/unknown_field.rs:5:21
  |
5 | #[shorthand(disable(garbage))]
  |                     ^^^^^^^
